### PR TITLE
update metadata for releasing w/project renames

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

File: bottomless-cli/Cargo.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ version = "0.1.14"
 edition = "2021"
 license = "MIT"
 keywords = ["libsql", "sqlite", "s3", "cli", "replication"]
-repository = "https://github.com/libsql/sqld"
+repository = "https://github.com/libsql/libsql"
```
Comment (LucioFranco) [inline bottomless-cli/Cargo.toml:7]: Well get this fixed eventually as well as the org is wrong now too, but well do that.

Comment (ashleygwilliams) [reply bottomless-cli/Cargo.toml]: fixed while i was waiting, heh

**Review by MarinPostma — COMMENTED** [review]
I don't see the old insta snapshot files being removed, I think the rename left them behind?

**Comment by ashleygwilliams** [comment]
oh huh @MarinPostma that looks correct actually. i'll clean that up. additionally, there's a test i can't run locally due to a lack of a runtime dep i think- so that's the origin of the test failure i believe. will try to sort that but welcome any pointers!

**Comment by ashleygwilliams** [comment]
(i'll also manage the conflict)

**Comment by ashleygwilliams** [comment]
(hopping in a call now but will fix my rebase botch when i'm out)

**Comment by MarinPostma** [comment]
Thanks for the effort @ashleygwilliams 😄

**Pull request merged**